js/bitrue.js

```diff
@@ -1676,6 +1676,42 @@ module.exports = class bitrue extends Exchange {
         return this.parseTransactions (data, currency);
     }
 
+    async fetchTransactionFees (codes = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitrue#fetchTransactionFees
+         * @description fetch transaction fees
+         * @see https://github.com/Bitrue-exchange/Spot-official-api-docs#general-endpoints
```

Comment:
Should be https://github.com/Bitrue-exchange/Spot-official-api-docs#exchangeInfo_endpoint